utkdigitalinitiatives/utk-archivesspace-local
# Move record innards to right location.

**JIRA Ticket**: [AS-69](https://jirautk.atlassian.net/browse/AS-69)

# What does this Pull Request do?

This makes "expanded by default" accordion work for ArchivesSpace.

# What's new?

A month or so ago Special Collections asked us to make Collections Finding Aids be expanded by default in ArchivesSpace rather than collapsed.  While we did the work for this in our plugin, we didn't put the file in the right location.  Therefore, even though it works in production, it's not working on localhost, and in theory could be lost in a server move / service restore. 

# How should this be tested?

Do one of these two things:

1. Install this branch of our local plugin in your local dev environment and make sure the collection view is expanded by default (like it is on production currently).
2. If you want to test with [Docker](https://github.com/utkdigitalinitiatives/utk_archivesspace_docker), switch the .env branch variable value to "AS-69" and do a docker-compose up --build -d.  Keep in mind that you'll need to give Docker time to complete an import job (you can monitor jobs in [staff view](http://localhost:8080/jobs)).  Just make sure one job is archived before you test.

# Additional Notes:
Once this is merged, nothing needs to happen in production for now. 

Also, we only need one reviewer. I added a bunch of people since @CanOfBees is out.

# Interested parties
@utkdigitalinitiatives/digital-initiatives-developers
